tests/cephadm/test_admin_add_labels.py

```diff
@@ -0,0 +1,65 @@
+from ceph.ceph_admin import CephAdmin
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def _add_host_label(instance):
+    """
+    Method add multiple labels to cephadmin node
+    Args:
+        instance: cluster instance
+    Returns:
+        1 -> Fail
+        0 -> Pass
+    """
+
+    node = instance.cluster.get_nodes()[0].hostname
+    cmd = (
+        "cephadm shell ceph orch host add {} node-exporter ".format(node)
+        + "crash osd installer grafana prometheus alertmanager"
+    )
+    try:
+        out, _ = instance.installer.exec_command(cmd, sudo=True)
+        return 0
+    except Exception as e:
+        log.error(e)
+
+
+def run(ceph_cluster, **kw):
+
+    config = kw.get("config")
+
+    # Checking files in path /etc/ceph before adding multiple lable
+    instance = CephAdmin(cluster=ceph_cluster, **config)
+    out, _ = instance.installer.exec_command(cmd="ls /etc/ceph", sudo=True)
+    files_before = out.split("\n")
+    log.info(
+        "File exist in path /etc/ceph/ before adding multiple lables {}".format(
+            files_before
+        )
+    )
+
+    # Adding multiple roles in same admin node
+    label = _add_host_label(instance)
+    if label == 1:
+        log.error("Failed to add multiple roles in same admin node")
+        return 1
+
+    # Checking cluster health and files in path /etc/ceph after multiple lable
+    health, _ = instance.installer.exec_command(
+        cmd="cephadm shell ceph -s | grep health:", sudo=True
+    )
+    out, _ = instance.installer.exec_command(cmd="ls /etc/ceph", sudo=True)
```

Comment:
`/etc/ceph` is used multiple time here, better to create const variable and use

Reply:
Updated

Reply:
@MohitBis create separate variable for `/etc/ceph` and use it in all places where this dir is used. Like - 
```
ceph_dir = "/etc/ceph"
.
.
out, _ = instance.installer.exec_command(cmd=f"ls {ceph_dir}", sudo=True)
.
.
.
log.info(
    f"File exist in path '{ceph_dir}' after adding multiple lables '{files_after}'"
)
```

Reply:
Updated